diff --git a/lib/ec2/platform/base/constants.rb b/lib/ec2/platform/base/constants.rb
index 8aaa7db..e1beab0 100644
--- a/lib/ec2/platform/base/constants.rb
+++ b/lib/ec2/platform/base/constants.rb
@@ -16,11 +16,11 @@ module EC2
     module Base
       module Constants
         module Bundling
-          EC2_HOME = ENV["EC2_AMITOOL_HOME"] || ENV["EC2_HOME"]
-          EC2_X509_CERT = File.join(EC2_HOME.to_s, '/etc/ec2/amitools/cert-ec2.pem')
-          EC2_X509_GOV_CERT = File.join(EC2_HOME.to_s, '/etc/ec2/amitools/cert-ec2-gov.pem')
-          EC2_X509_CN_NORTH_1_CERT = File.join(EC2_HOME.to_s, '/etc/ec2/amitools/cert-ec2-cn-north-1.pem')
-          EC2_MAPPING_FILE = File.join(EC2_HOME.to_s, '/etc/ec2/amitools/mappings.csv')
+          EC2_AMITOOL_SHARE = '/usr/share/ec2-ami-tools'
+          EC2_X509_CERT = File.join(EC2_AMITOOL_SHARE.to_s, '/certs/cert-ec2.pem')
+          EC2_X509_GOV_CERT = File.join(EC2_AMITOOL_SHARE.to_s, '/certs/cert-ec2-gov.pem')
+          EC2_X509_CN_NORTH_1_CERT = File.join(EC2_AMITOOL_SHARE.to_s, '/certs/cert-ec2-cn-north-1.pem')
+          EC2_MAPPING_FILE = File.join(EC2_AMITOOL_SHARE.to_s, '/mappings.csv')
           EC2_MAPPING_URL = 'https://ec2-downloads.s3.amazonaws.com/mappings.csv'
           DESTINATION = '/tmp'
         end
